dotnet/src/SemanticKernel/CoreSkills/TimeSkill.cs

```diff
@@ -52,6 +52,16 @@ public string Date()
         return DateTimeOffset.Now.ToString("D", CultureInfo.CurrentCulture);
     }
 
+    /// <summary>
+    /// Get the current date
+    /// </summary>
+    /// <example>
+    /// {{time.today}} => Sunday, 12 January, 2031
+    /// </example>
+    /// <returns> The current date </returns>
+    [SKFunction("Get the current date")]
+    public string Today() => this.Date();
```

Comment:
Not sure we want both, as this will manifest as twice the input (tokens) when surfacing this skill to the AI via Planner. @lemillermicrosoft thoughts? What's the perspective on aliased functions with identical descriptions?

Reply:
Closing this thread to resolve separately